code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1554,13 +1554,13 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 		var/datum/data/record/R = find_record("name", perpname, GLOB.data_core.security)
 		if(R && R.fields["criminal"])
 			switch(R.fields["criminal"])
-				if("*Execute*")
+				if(SEC_RECORD_STATUS_EXECUTE)
 					threatcount += 7
-				if("*Arrest*")
+				if(SEC_RECORD_STATUS_ARREST)
 					threatcount += 5
-				if("Incarcerated")
-					threatcount += 2
-				if("Parolled")
+				if(SEC_RECORD_STATUS_INCARCERATED)
+					threatcount += 5
```

Comment:
Might want to check this to see if beepsky doesn't go ham on prisoners who are in cells already

Reply:
Indeed, now I remember realising that previously, already forgot it. Changed it back to 2 as it was. Thanks!